docs/how-to/purge-data-on-shadow-table.rst

```diff
@@ -0,0 +1,30 @@
+==========================================
+Purge data on shadow tables
+==========================================
+
+The purge steps is disable by default and plans to run before proceeding to actual upgrade steps.
+This can be turn on with the `--purge`` flag.
+
+This purge step is a performance optimisation, delete data from the shadow tables in nova database. The behavior is equal to run juju action `purge` on nova-cloud-controller unit, which can help to reduce the size of the db.
```

Comment:
Note: rst uses double backticks for preformatted text - there are a couple of locations in this doc with single backticks.